tests/test_api.py

```diff
@@ -207,12 +207,17 @@ def test_get_block_content(self, mock_get_children_contents, mock_get_single_blo
         self.assertEqual(items, content_items)
 
     @ddt.data(
-        'This is content.',
-        ''
+        ('This is content.', True),
+        ('', True),
+        ('This is content.', False),
+        ('', False),
     )
+    @ddt.unpack
+    @patch('learning_assistant.toggles._is_learning_assistant_waffle_flag_enabled')
     @patch('learning_assistant.api.get_block_content')
-    def test_render_prompt_template(self, unit_content, mock_get_content):
+    def test_render_prompt_template(self, unit_content, enable_content, mock_get_content, mock_flag_enabled):
```

Comment:
Clarification: could you clarify why there is both enable_content and mock_flag_enabled? Thanks!

Reply:
`enable_content` is the value being set for the waffle flag, whereas `mock_flag_enabled` is a mock for the function `_is_learning_assistant_waffle_flag_enabled`. Those names are definitely a bit confusing so I will rename them!